server/models/issue.js

```diff
@@ -0,0 +1,61 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+const IssueSchema = new Schema(
+  {
+    description: {
+      type: String,
+      trim: true,
+      required: true,
+    },
+    createdBy: {
+      type: Schema.Types.ObjectId,
+      ref: 'User',
+      required: true,
+    },
+    modifiedBy: {
```

Comment:
either make it an array or remove completely. it can be modified multiple times

Reply:
The idea here is to show who was the last person that modified the issue.